docs/source/release.rst

```diff
@@ -5,6 +5,48 @@
 Change Log
 ==========
 
+Release 2.10 - xx/xx/xxxx
+-------------------------
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*  Added deduplication for evolutionary search.
+*  Fixed the racing issue in RL strategy submit model.
+*  Fixed an issue introduced by the trial recovery feature.
+*  Fixed import error of ``Lightning`` in nas.
+
+Compression
+"""""""""""
+
+*  #3514
+*  Fixed the bug that speedup ``rand_like_with_shape`` is easy to overflow when ``dtype=torch.int8``.
+*  Fixed the bug when calling ``torch._C.parse_schema`` in pytorch < 1.9.0.
+*  Fixed the propagation error with view tensors in speedup.
+
+Hyper-parameter optimization
+""""""""""""""""""""""""""""
+
```

Comment:
could add one more: Supported rerunning (resuming) the interrupted trials induced by the termination of an NNI experiment when resuming this experiment.